src/Resources/config/services.xml

```diff
@@ -15,8 +15,9 @@
             <tag name="doctrine_mongodb.odm.event_subscriber" />
         </service>
 
-        <service id="algolia_client" class="AlgoliaSearch\Client" public="false" lazy="true">
```

Comment:
At the time, I didn't want to make the client public because I wanted to make this package "agnostic".
This is forcing people to override the container configuration to make it public.

Where do we stand about this package being "agnostic"?

I'm incline to make this client public, it will be easier to use.